### PR TITLE
created Home Page of note Taking app

### PR DESCRIPTION
**Pull request by Rockyraven**
notehomepage

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+import { HomePage } from './pages/homepage/HomePage'
+
 
 export const App = () => {
-  return (
-    <div>App</div>
+  return (<>
+  <Navbar/>
+    {/* <Routes> */}
+        {/* <Route path='/' element = {<HomePage/>}/> */}
```
Comment (vermasavita) [inline src/App.js]: Delete this comment if not needed.

File: src/component/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export const Footer = () => {
+    return (
+        <footer className='footer-container'>
+            <p>Made with ❤️ by Rocky</p>
+            <p>Copyright &copy; 2022 Quick-Quiz, INC. All Right Reserved</p>
+            <div className="social-link-container">
+
+            <a href="https://www.linkedin.com/in/rocky-abb69921b/"><i class="fa fa-linkedin"></i></a>
+            <a href="https://twitter.com/ImRocky7277" target="_blank"><i className="fa fa-twitter"></i></a>
```
Comment (vermasavita) [inline src/component/footer/Footer.js]: Instead of anchor tag you can use Link

File: src/component/navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+// import { Link } from 'react-router-dom'
```
Comment (vermasavita) [inline src/component/navbar/Navbar.js]: Delete the commented code

File: src/component/navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+// import './navbar.css'
+import './navbar.css'
+
+
+export const Navbar = () => {
+  return (
+    <nav className="nav-heading">
+    <a href="/"><h1 className="home-heading">QuickNote</h1></a>
```
Comment (vermasavita) [inline src/component/navbar/Navbar.js]: Same here, you can use <Link> tag

**Review by vermasavita — APPROVED** [review]

**Comment by rahulmistry751** [comment]
Well structured code.

**Review by rahulmistry751 — APPROVED** [review]

**Pull request merged**